Wiki/configuration.md

```diff
@@ -0,0 +1,29 @@
+## Configuration
+
+Trilium supports configuration via a file named `config.ini` and environment variables. Please review the file named `config-sample.ini` in the [Notes](https://github.com/TriliumNext/Notes) repository to see what values are supported.
```

Comment:
Any reason not to link directly to the `config-sample.ini` in the repo? The potential for a future broken link is similar to the potential for the file to move or be renamed and thus this info without the link being outdated.

Reply:
I figured that the config's keys would evolve over time so a permalink might have older keys...but I think that a simple link would be fine :)